build.sbt

```diff
@@ -176,11 +177,20 @@ lazy val benchmarks = Project(id = "benchmarks", base = file("benchmarks"))
     scalaVersion := Dependencies.Versions.CrossScalaForLib.head,
     (publish / skip) := true)
 
+// Config to allow only building scaladocs for runtime module but in/from the docs module
+val Runtime = config("runtime")
```

Comment:
name it something like grpc-runtime? "runtime" sounds like it could be something standard in sbt

Reply:
done